webapp/channels/src/actions/views/create_comment.tsx

```diff
@@ -197,7 +208,7 @@ export function onSubmit(
             }
         }
 
-        return dispatch(submitPost(channelId, rootId, draft, options.afterSubmit, schedulingInfo, options.afterOptimisticSubmit));
+        return dispatch(submitPost(channelId, rootId, draft, options.afterSubmit, schedulingInfo, options.afterOptimisticSubmit, options.keepDraft));
```

Comment:
Maybe we should start thinking about passing here just the options object :sweat_smile:

Reply:
Done.